### DISCUSSION TITLE
Project Tracker sample outdated?

### DISCUSSION THREAD

**Discussion by Conway6288** [MarimerLLC/csla#4086] [Questions]
When configuring the DalEfCore of the ProjectTracker sample, should the AddScoped call to PTrackerContext be AddDbContext? 

https://github.com/MarimerLLC/csla/blob/main/Samples/ProjectTracker/ProjectTracker.DalEfCore/ConfigurationExtensions.cs

**Comment by rockfordlhotka** [comment]
I've been working on updating the CSLA 9 version of ProjectTracker, though even that's out of date because of recent changes to CSLA 9.

I know the CSLA 8 one was pretty out of date.
